sound/soc/sof/Makefile

```diff
@@ -27,9 +27,12 @@ obj-$(CONFIG_SND_SOC_SOF_ACPI_DEV) += snd-sof-acpi.o
 obj-$(CONFIG_SND_SOC_SOF_OF_DEV) += snd-sof-of.o
 obj-$(CONFIG_SND_SOC_SOF_PCI_DEV) += snd-sof-pci.o
 
+# Build the SOF client drivers only if the SOF core is enabled
```

Comment:
look good but the commit message has a couple of typos "coce, the the etc". Also, I wonder if we can make make SND_SOC_SOF_CLIENT depend on SND_SOC_SOF in Kconfig instead?

Reply:
@ranj063, I think the solution in #3322 is better, closing this, thanks for the review!